AlinaFirsova/Homework 1/style.css

```diff
@@ -0,0 +1,19 @@
+front_main{ 
+    font-size: 100%; 
+    font-family: Francois One, bold; 
+    color: white;
+   }
+front_add{ 
+    font-size: 80%; 
+    font-family: Francois One, bold; 
+    color: white;
+   }
+
+.center{
+   padding-top: 15%;
+ padding-left: 27%;
+padding-bottom: 15%;
+ padding-right: 27%;
+margin: 0 auto; 
+text-align: center;"    
```

Comment:
@AlinaFirsova Pay attention to inappropriate simbols in css file. `"` double quote here